Python/WebCrawler/Readme.MD

```diff
@@ -0,0 +1,11 @@
+## This is a web crawler for grabbing links from a website 
+
+# The code is built in Python
+
+# A sample output is like this while scrapping a webpage of a GeekForGeeks Website
```

Comment:
you didn't update the readme. You mentioned geeksforgeeks while image sample shows another website.